src/components/component.js

```diff
@@ -1,3 +1,20 @@
+/**
+ * Component class is a parent for classes that implement sendKeys and click. For example input, textarea,
+ * select and checkbox.
+ *
```

Comment:
Component class is a parent of ALL the classes in Spectangular. Besides click and sendKeys it also implements the basic API all components in spectangular have. Like componentEl selection based on selector, rootEl, whenVisible, whenInvisible etc.